src/main/java/org/kaazing/netx/ws/internal/WebSocketTransition.java

```diff
@@ -17,6 +17,8 @@
 
 public enum WebSocketTransition {
     RECEIVED_UPGRADE_RESPONSE,
+    RECEIVED_CONTINUATION_FRAME,
+    SEND_CONTINUATION_FRAME,
     RECEIVED_CLOSE_FRAME,
     SEND_CLOSE_FRAME,
     RECEIVED_PING_FRAME,
```

Comment:
Perhaps this should be RECEIVE / SEND or RECEIVED / SENT but not RECEIVED / SEND ?
The transitions are active verbs (not past tense), right?